docs/source/docker/getting_started/first_steps.rst

```diff
@@ -302,6 +302,35 @@ To make it easier for you to understand and discuss the dataset we put the essen
 an automatically generated PDF report.
 Sample reports can be found on the `Lightly website <https://lightly.ai/analytics>`_.
 
+
+.. _ref-docker-runs:
+
+Live View of Docker Status
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can get a live status update of the currently running docker runs through 
+the `cloud platform <https://app.lightly.ai>`_. 
+
+To use the new feature simply follow the steps:
+
+#. Make sure you have the latest docker version installed
+   (see :ref:`ref-docker-download-and-install`) 
+#. Open a browser and navigate to the `Lightly Platform <https://app.lightly.ai>`_
+#. In the navigation menu on the top click on **My Docker Runs**
+#. Once you start the Lightly Docker you should see the dashboard of the current
+   run. Please make sure that you use the same token for the docker run as you
+   use in the dashboard.
```

Comment:
find in the dashboard? 

because you're not really using a token in the web-app